src/main/java/org/traccar/geocoder/AmapGeocoder.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2012 - 2017 Anton Tananaev (anton@traccar.org)
```

Comment:
Please fix the header.

Reply:
OK, the code here I copied directly from another class in the project. I'll rework it.